node/node.go

```diff
@@ -119,7 +121,11 @@ func NewNode(config cfg.Config, privValidator *types.PrivValidator, clientCreato
 	consensusReactor := consensus.NewConsensusReactor(consensusState, fastSync)
 
 	// Make p2p network switch
-	sw := p2p.NewSwitch(config.GetConfig("p2p"))
+	p2pConfig := viper.New()
+	if config.IsSet("p2p") { //TODO verify this necessary, where is this ever set?
```

Comment:
yup, I get the intention. I was just wondering if this had become antiquated, can remove the comment if we're certain we want to leave. Not currently used anywhere that I can find